src/communication/network_interface_cl.c

```diff
@@ -8124,6 +8124,285 @@ logtb_dump_trantable (FILE * outfp)
+ */
+// *INDENT-OFF*
+int
+mmon_get_module_info_summary (int module_count, std::vector<MMON_MODULE_INFO> & module_info)
+// *INDENT-ON*
+{
+#if defined(CS_MODE)
+  char *buffer, *ptr, *temp_str;
+  int bufsize = 0;
+  OR_ALIGNED_BUF (OR_INT_SIZE) a_request;
+  char *request = OR_ALIGNED_BUF_START (a_request);
+  OR_ALIGNED_BUF (OR_INT_SIZE + OR_INT_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  int req_error, ival;
+  int error = NO_ERROR;
+
+  ptr = or_pack_int (request, module_count);
+
+  req_error =
+    net_client_request2 (NET_SERVER_MMON_GET_MODULE_INFO_SUMMARY, request, OR_ALIGNED_BUF_SIZE (a_request),
+			 reply, OR_ALIGNED_BUF_SIZE (a_reply), NULL, 0, &buffer, &bufsize);
+  if (req_error)
+    {
+      assert (er_errid () != NO_ERROR);
+      error = er_errid ();
+    }
+  else
+    {
+      ptr = reply;
+      ptr = or_unpack_int (ptr, &ival);
+      ptr = or_unpack_int (ptr, &ival);
+      error = (int) ival;
+    }
+
+  if (error == NO_ERROR)
+    {
+      module_info.resize (module_count);
+
+      ptr = buffer;
+
+      // *INDENT-OFF*
+      for (auto &m_info : module_info)
+        {
+          ptr = or_unpack_string (ptr, &temp_str);
+          memcpy (m_info.name, temp_str, strlen (temp_str) + 1);
+          ptr = or_unpack_int64 (ptr, (int64_t *) &(m_info.stat.cur_stat));
+        }
+      // *INDENT-ON*
+    }
+  free_and_init (buffer);
+
+  return error;
+#else /* CS_MODE */
+  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_NOT_IN_STANDALONE, 1, "memmon");
+  return ER_NOT_IN_STANDALONE;
+#endif /* !CS_MODE */
+}
+
+/*
+ * mmon_get_tran_info -
+ *
+ * return: cubrid error
+ *
+ *   tran_count(in): the number of active transactions to print information
+ *   tran_info(in/out): save memory usage information of the active transactions
+ */
+int
+mmon_get_tran_info (int tran_count, MMON_TRAN_INFO & tran_info)
+{
+#if defined(CS_MODE)
+  char *buffer, *ptr;
+  int bufsize = 0;
+  OR_ALIGNED_BUF (OR_INT_SIZE) a_request;
+  char *request = OR_ALIGNED_BUF_START (a_request);
+  OR_ALIGNED_BUF (OR_INT_SIZE + OR_INT_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  int req_error, ival;
+  int error = NO_ERROR;
+
+  ptr = or_pack_int (request, tran_count);
+
+  req_error =
+    net_client_request2 (NET_SERVER_MMON_GET_MODULE_INFO_SUMMARY, request, OR_ALIGNED_BUF_SIZE (a_request),
+			 reply, OR_ALIGNED_BUF_SIZE (a_reply), NULL, 0, &buffer, &bufsize);
+  if (req_error)
+    {
+      assert (er_errid () != NO_ERROR);
+      error = er_errid ();
+    }
+  else
+    {
+      ptr = reply;
+      ptr = or_unpack_int (ptr, &ival);
+      ptr = or_unpack_int (ptr, &ival);
+      error = (int) ival;
+    }
+
+  if (error == NO_ERROR)
+    {
+      ptr = buffer;
+      ptr = or_unpack_int (buffer, (int32_t *) & (tran_info.num_tran));
+
+      tran_info.tran_stat.resize (tran_info.num_tran);
+
+      // *INDENT-OFF*
+      for (auto &tran_stat : tran_info.tran_stat)
```

Comment:
```suggestion
      for (const auto &tran_stat : tran_info.tran_stat)
```
Let's make all const if possible.

Reply:
oops, it's impossible. Sorry.